test/demo/utils/toastUtils.ts

```diff
@@ -0,0 +1,29 @@
+import {ElementHandle} from 'puppeteer';
+import {Identity, Window} from 'hadouken-js-adapter';
+
+import {SERVICE_IDENTITY} from '../../../src/client/internal';
+
+import {OFPuppeteerBrowser} from './ofPuppeteer';
+import {fin} from './fin';
+
+export function getToastIdentity(sourceApp: Identity, notificationId: string): Identity {
+    return {uuid: SERVICE_IDENTITY.uuid, name: `Notification-Toast:${sourceApp.uuid}:${notificationId}`};
+}
+
+export async function getToastWindow(sourceApp: Identity, notificationId: string): Promise<Window | undefined> {
```

Comment:
Standardise on using `null` for the "no such window" return value? Same as `getToastCards`, and most other places.

Reply:
Sure. I was torn between the two. Would you rather this returns null or have the others return undefined?

Reply:
The `.find` in getToastWindow returns `Window | undefined` and I figured it's easier to just have that return directly and change the null in getToastCards to undefined, so they match now.